bin/migrate-oats-data/menu/commands/import_all.py

```diff
@@ -38,18 +40,23 @@ def import_all(console, args):
         console.log("Init NOIs:")
         init_notice_of_intents(batch_size=import_batch_size)
 
-        # TODO Liam question: which process_documents_noi or process_noi_documents is the correct one to keep?
-        # console.log("Processing NOI specific documents:")
-        # process_documents_noi(batch_size=import_batch_size)
+        # OATS -> ALCS documents = process_documents & process_documents_noi. Must take place in order to have usable rows in ALCS documents
+        # ALCS documents -> application_documents/noi_documents = process_application_documents & process_noi_documents
+        console.log("Importing NOI specific OATS documents into ALCS:")
+        process_documents_noi(batch_size=import_batch_size)
```

Comment:
Do I get it right that the process_documents_noi is responsible for importing oats.documents that belong to NOI to alcs.documents?

Reply:
That is correct

Reply:
Lets then update the naming to: process_documents_noi -> import_oats_noi_documents, process_noi_documents-> link_alcs_documents_to_noi_documents. And the same style for applications. That way it will be clear and we do not need comments at all